src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/HtmlController.kt

```diff
@@ -76,6 +76,8 @@ class HtmlController(@Autowired val moveService: MoveService, @Autowired val jou
     fun journeys(
             @ModelAttribute(name = DATE_ATTRIBUTE) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) startOfMonth: LocalDate,
             @ModelAttribute(name = SUPPLIER_ATTRIBUTE) supplier: Supplier,
+            @ModelAttribute(name = "locationName") locationName: String,
```

Comment:
It's not immediately obvious but these are flash attributes, perhaps I could rename them?

Reply:
I think that's a good idea

Reply:
Have addressed.  This will show up in next PR.